docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -132,7 +132,7 @@ Amazon calls a virtual private server, a **virtual server** or **Amazon EC2 inst
 3. Click on the blue `Launch Instance` button.
 
 - `Select` **Ubuntu Server 18.04 LTS (HVM), SSD Volume Type**
-- Ensure `General purpose` + `t2.small` is `checked`. **NOTE:** `t2.small` is the smallest instance type in which Strapi runs. `t2.nano` and `t2.micro` **DO NOT** work.
+- Ensure `General purpose` + `t2.small` is `checked`. **NOTE:** `t2.small` is the smallest instance type in which Strapi runs. `t2.nano` and `t2.micro` **DO NOT** work. At the moment, deploying the Strapi Admin interface requires more than 1g of RAM. Therefore, **t2.small** or larger instance is needed.
```

Comment:
Ensure `General purpose` + `t2.small` is `checked`. **NOTE:** `t2.small` is the smallest instance type in which Strapi can run.

Reply:
That does seem clearer.  Thank you. Fixed.

Reply:
Note that this is not true. Strapi runs perfectly fine on t2.micro. https://github.com/strapi/strapi/issues/3374

Reply:
> Note that this is not true. Strapi runs perfectly fine on t2.micro. #3374

@jsve the amount of people I talk to, and my own testing assures that it does not right now with the main issue being node-sass

Reply:
@derrickmehaffy @jsve - You both are kinda right.  Strapi cannot **_deploy_** on a t2.micro but needs to deploy on a t2.small or above.  But once deployed, it will run on a t2.micro.  And the problem is node-sass... Which @soupette is addressing.

Having said that, there have been cases were t2.micro isn't sufficient for running it either so we're recommending t2.small.